src/oidcc_cowboy_callback.erl

```diff
@@ -83,6 +85,7 @@
 %%   <li>`check_peer_ip' - check if the client IP is the same as before the
 %%     authorization request</li>
 %%   <li>`retrieve_userinfo' - whether to load userinfo from the provider</li>
+%%   <li>`scopes' - list of scopes to use in token request (if not present in Req); defaults to `[openid]'</li>
```

Comment:
should we default to `openid` when the user does not specify the scopes? I'd rather have an error and the user has to think about it and put the correct scopes in. What do you think?

Reply:
I‘d prefer the error since the default could‘ve been changed by the user without knowing to do it in both places.